src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/Consent.cshtml

```diff
@@ -10,55 +10,4 @@
     <a href="@Model.BackUrl" class="govuk-back-link">Back</a>
 }
 
-<partial name="_ErrorSummaryNext" for="@Model.Errors" />
-
-<div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-
-        <h1 class="govuk-heading-l">
-            Permission to share details
-        </h1>
-        <form method="post">
-            <p>You must get permission from the people who need support to share their details.</p>
-            <p>Those details include:</p>
-            <ul class="govuk-list govuk-list--bullet">
-                <li>who to contact</li>
-                <li>ways to contact them</li>
-                <li>why they need help</li>
-            </ul>
-
-            <div class="govuk-form-group @(Model.HasErrors ? "govuk-form-group--error" : "")">
-                <fieldset class="govuk-fieldset">
-                    <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
-                        <h2 class="govuk-fieldset__heading">
-                            Do you have permission to share their details?
-                        </h2>
-                    </legend>
-                    <div class="govuk-radios govuk-radios--inline" data-module="govuk-radios">
-                        @* todo: add aria-described-by to error *@
-                        @{
-                            var error = Model.Errors.GetErrorIfTriggered();
-                        }
-                        <partial name="_ErrorMessageNext" model="error"/>
-                        <div class="govuk-radios__item">
-                            <input class="govuk-radios__input" id="consent-yes" name="Consent" type="radio" value="true" @(Model.Consent == true ? "checked" : "")>
-                            <label class="govuk-label govuk-radios__label" for="consent-yes">
-                                Yes
-                            </label>
-                        </div>
-                        <div class="govuk-radios__item">
-                            <input class="govuk-radios__input" id="consent-no" name="Consent" type="radio" value="false" @(Model.Consent == false ? "checked" : "")>
-                            <label class="govuk-label govuk-radios__label" for="consent-no">
-                                No
-                            </label>
-                        </div>
-                    </div>
-                </fieldset>
-            </div>
-            <button type="submit" class="govuk-button" data-module="govuk-button">
-                Continue
-            </button>
-        </form>
-    </div>
-
-</div>
+<partial name="_RadiosPage" Model="@Model" />
```

Comment:
This line above...

```Model.Errors.ErrorIdToHtmlElementId = errorId => "consent-yes";```

is a callback to map from an errorId to the id of the element related to the error.

The id is used to populate the error message link in the error summary, so that when it's clicked, it focuses on the right input element.

As all errors on this page should link to the Yes radio button, it can ignore the errorId and just supply the id of the Yes radio button, which I _think_ will be `radio-True`.